install-appliance.md

```diff
@@ -22,45 +22,70 @@ The Anbox Cloud Appliance is available through the AWS Marketplace and as a snap
 <a name="install-aws"></a>
 ### Install from AWS Marketplace
 
-Installing the Anbox Cloud Appliance through the AWS Marketplace simplifies the installation and deployment process and allows billing to be handled directly through AWS. The following steps guide you through all relevant steps to deploy the Anbox Cloud Appliance in your AWS account.
+Installing the Anbox Cloud Appliance through the AWS Marketplace simplifies the installation and deployment process and allows billing to be handled directly through AWS.
 
-#### Choose an instance architecture
-
-First, decide if you want to deploy the Anbox Cloud Appliance on an x86 or Arm instance architecture. AWS supports both.
-
-The decision should factor in the following aspects:
+AWS supports running the Anbox Cloud Appliance on an x86 or Arm instance architecture. Before installing the appliance, decide which architecture you want to use. The decision should factor in the following aspects:
 
 * GPUs are currently available for x86. NVIDIA GPUs will only become available for Arm instances [later in 2021](https://aws.amazon.com/blogs/machine-learning/aws-and-nvidia-to-bring-arm-based-instances-with-gpus-to-the-cloud/).
 * Not all Android applications support the x86 ABI. Therefore, some applications can run only on Arm.
 
-Once you have decided for an instance architecture, go to the corresponding marketplace page:
+For detailed information about the offering, see the following pages on the AWS Marketplace:
 
-* [Anbox Cloud Appliance for x86](https://aws.amazon.com/marketplace/pp/prodview-3lx6xyaapstz4)
 * [Anbox Cloud Appliance for Arm](https://aws.amazon.com/marketplace/pp/prodview-aqmdt52vqs5qk)
+* [Anbox Cloud Appliance for x86](https://aws.amazon.com/marketplace/pp/prodview-3lx6xyaapstz4)
+
+The following instructions guide you through all relevant steps to deploy the Anbox Cloud Appliance in your AWS account. For additional information, see the [AWS documentation](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/launching-instance.html) about launching an instance.
+
+#### 1. Start the launch wizard
```

Comment:
Don't you need to subscribe to the marketplace offering before?

Reply:
No. If you start with subscribing, you will click on "Continue to configure" next and end up in the wizard that doesn't let you add storage.